ci-operator/step-registry/rosa/aws/sts/hypershift/capi/rosa-aws-sts-hypershift-capi-workflow.metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+	"path": "rosa/aws/sts/hypershift/capi/rosa-aws-sts-hypershift-capi-workflow.yaml",
```

Comment:
For you have achieved the new step `cucushift-hypershift-extended-capi-init` rather than `rosa-cluster-provision`, I think adding the workflow to where you do capi cluster provision could be better.

Reply:
got it. Would you prefer to move the steps to the "rosa" directory or to move the workflow to the "hypershift-extended" directory?

Reply:
I am not sure which is the core scope, if for capi, maybe cucushift-hypershift-extended-capi-clear could be better, if for rosa, rosa-cluster-capi could be better.

Reply:
this is triggered by capi, the core test is to test capi process to trigger a rosa hcp.  And it is only for hcp now. So if you agree that we move all of steps to rosa side, it would be rosa-aws-sts-hypershift-capi.
Or let's keep it in hypershift-extended temporarily, and then consider how to optimize it with other features together later.

Reply:
I checked the cilium ci. It is similar with this one. The steps are more related to hypershift side for both rosa/self-managed hypershift. But I'd like to add workflow to rosa dir because it is for rosa hcp with the feature. 

So I suggest following the same way to build capi workflow, since the workflow is more about rosa hcp and steps are more related to capi things.

Reply:
Got it. Let us keep the same way.

Reply:
thanks